io-engine/src/core/fault_injection/mod.rs

```diff
@@ -34,7 +34,7 @@ pub enum FaultDomain {
     NexusChild,
     /// Fault injection on block device abstraction level.
     BlockDevice,
-    ///
+    /// TODO
```

Comment:
Did you plan to do this and missed by chance ?

Reply:
Now it's required to add comment text, but I don't know what to write for it, so left it like so